apps/create-vesting/src/app/page.tsx

```diff
@@ -1,126 +1,162 @@
 "use client";
 
-import {
-  type CoWHookDappActions,
-  type HookDappContext,
-  initCoWHookDapp,
-} from "@cowprotocol/hook-dapp-lib";
-
 import {
   Input,
   PeriodWithScaleInput,
   ButtonPrimary,
   ContentWrapper,
   TokenAmountInput,
   Wrapper,
+  HookDappContextAdjusted,
 } from "@bleu/cow-hooks-ui";
+import { Token } from "@uniswap/sdk-core";
 
-import { useEffect, useState } from "react";
-
+import { useCallback, useMemo } from "react";
+import { useIFrameContext, Spinner } from "@bleu/cow-hooks-ui";
 import { Form } from "@bleu/ui";
 import { zodResolver } from "@hookform/resolvers/zod";
 import { useForm } from "react-hook-form";
 
 import { createVestingSchema, periodScaleOptions } from "#/utils/schema";
 
-//will be used
-import { scaleToSecondsMapping } from "#/utils/scaleToSecondsMapping";
+import { useGetHooksTransactions } from "#/hooks/useGetHooksTransactions";
+import { useRouter } from "next/navigation";
+import { useReadTokenContract } from "#/hooks/useReadTokenContract";
+import { vestingFactoriesMapping } from "#/utils/vestingFactoriesMapping";
+import { WeirollCheckbox } from "#/components/WeirollCheckbox";
```

Comment:
```suggestion
import { VestAllBuyAmountCheckbox } from "#/components/WeirollCheckbox";
```

imo weiroll is just the interface/backend that we use to do it, but the component name should refer to the functionality